roxctl/deployment/check/check.go

```diff
@@ -187,10 +203,18 @@ func (d *deploymentCheckCommand) Check() error {
 }
 
 func (d *deploymentCheckCommand) checkDeployment() error {
-	deploymentFileContents, err := os.ReadFile(d.file)
-	if err != nil {
-		return errors.Wrapf(err, "could not read deployment file: %q", d.file)
+	var deploymentFileContents []byte
+	for _, file := range d.files {
+		fileContents, err := os.ReadFile(file)
+		if err != nil {
+			return errors.Wrapf(err, "could not read deployment file: %q", file)
+		}
+		if len(deploymentFileContents) > 0 {
+			deploymentFileContents = append(deploymentFileContents, "\n---\n"...)
```

Comment:
Haven't seen a test case for it, so might be worthwhile checking whether two separators in a row are seen as valid, i.e.
```yaml
something: test
---
---
another-thing: test
```

Reply:
The only way that would happen is if the user added an empty yml file, is that a valid use-case or should we throw an Error? eg: `roxctl deployment check --file deploy1.yml --file emptydeploy.yml --file deploy2.yml` resulting in
`
deploy1: test
---
---
deploy2: test
`

Reply:
I tested this and it is indeed a problem. If a file ends with a separator (`---`) it fails if there is another file after it. It works if checked individually or as the last file in the sequence.

This needs to be fixed before merging.

Reply:
My latest push fixes this by normalizing parsed YAML files